source/code/plugin/DockerApiClient.rb

```diff
@@ -86,7 +86,10 @@ def listContainers()
             containers = getResponse(request, true, false)
             if !containers.nil? && !containers.empty?
                 containers.each do |container|
-                    ids.push(container['Id'])
+                    labels = container['Labels']
+                    if !labels.nil? && !labels['io.kubernetes.pod.uid'].nil?
```

Comment:
Can you please make this a case-insensitive comparison, just in case... Thanks.

Reply:
Sure. Fixed

Reply:
Hi Rashmi - I was intending to say the label value (kubernetes.pod.uid) being case insensitive (not the property name)

Reply:
Sure.